src/table/TableBodyWrapper.jsx

```diff
@@ -35,8 +44,10 @@ export default function TableBodyWrapper({ tableData, constraints, selectionsAPI
             const cell = row[column.id];
             const value = cell.qText;
             const CellRenderer = columnRenderers[i];
-            return CellRenderer ? (
+            return (
               <CellRenderer
+                className={classes.tableCell}
+                stylingClassName={classes.tableCell}
```

Comment:
This is just a preference, but I think it was cleaner with only passing `className`, then using that in `withStyling`.

Reply:
i agree. will fix.